feature/security/gnsi/credentialz/tests/password_console_login/password_console_login_test.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package password_console_login
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/gnsi/credentialz"
+	tpb "github.com/openconfig/kne/proto/topo"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/binding"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	username          = "testuser"
+	password          = "i$V5^6IhD*tZ#eg1G@v3xdVZrQwj"
+	passwordVersion   = "v1.0"
+	passwordCreatedOn = 1705962293
```

Comment:
maybe a comment that: "this is unixepoch at may 1st 2025" (or whatever)"

Reply:
changed this to use `time.Now().Unix()` instead of static values